rosa_learning/deploying_application_workshop/cloud-experts-deploying-application-networking.adoc

```diff
@@ -0,0 +1,62 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-networking"]
+= Networking
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-networking
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-12-14
+
+The OSToy application uses intra-cluster networking to separate functions by using microservices.
+
+image::deploying-networking-arch.png[OSToy Diagram]
+
+In this workshop, there are at least two separate pods, each with its own service. One pod functions as the front end web application with a service and a publicly accessible route. The other pod functions as the backend microservice with a service object so that the front end pod can communicate with the microservice. 
```

Comment:
```suggestion
In this workshop, there are at least two separate pods, each with its own service. One pod functions as the front-end web application with a service and a publicly accessible route. The other pod functions as the backend microservice with a service object so that the front-end pod can communicate with the microservice. 
```

Reply:
I believe "front-end" should have the hyphen b/c it is being used as an adjective? 
Can't say the same for "backend" :(